storageprovider/csp/container_storage_provider.go

```diff
@@ -243,8 +243,8 @@ func (provider *ContainerStorageProvider) CreateVolume(name, description string,
 
 // CloneVolume clones a volume on the CSP
 // nolint : gocyclo
-func (provider *ContainerStorageProvider) CloneVolume(name, description, sourceID, snapshotID string, opts map[string]interface{}) (*model.Volume, error) {
-	log.Tracef(">>>>> CloneVolume with name: %v, description: %v, sourceID: %v, snapshotID %v", name, description, sourceID, snapshotID)
+func (provider *ContainerStorageProvider) CloneVolume(name string, description string, resize int64, sourceID string, snapshotID string, opts map[string]interface{}) (*model.Volume, error) {
```

Comment:
Why add string after name?  I would rename "resize" to "size" and put it after name, description, sourceID, snapshotID.

Reply:
Agreed, I will update it as per the suggestion.